doc/nrf/releases_and_maturity/releases/release-notes-2.9.0.rst

```diff
@@ -0,0 +1,661 @@
+
+* Updated:
+
+  * Thermostat cluster with support for scheduling and preset modes, like vacation, and home or away settings.
+  * Electric Vehicle Supply Equipment (EVSE) with support for user-defined charging preferences, like specifying the time when the car will be charged.
+  * Occupancy sensing cluster with features like radar, vision, and ambient sensing.
+  * Intermittently Connected Devices feature with enhancements for the Long Idle Time (LIT) and Check-In protocol.
+    With these enhancements, the state of this feature is changed from provisional to certifiable.
+
+Thread
+------
+
+* Added Kconfig options for configuring the MLE child update timeout, child supervision interval, and child supervision check timeout.
+
+Zigbee
+------
+
+* Updated:
+
+  * :ref:`nrfxlib:zboss` to v3.11.7.0 and platform v5.1.7 (``v3.11.6.0+5.1.7``).
+    They contain several fixes related to malfunctioning in a heavy traffic environment and more.
+    For details, see :ref:`zboss_changelog`.
+  * The :ref:`ZBOSS Network Co-processor Host <ug_zigbee_tools_ncp_host>` package to the new version v2.2.5.
+
+Wi-Fi
+-----
+
+* Updated the :ref:`wifi_regulatory_channel_rules` for some countries in the :ref:`ug_nrf70_developing_regulatory_support` documentation.
+
+Applications
+============
+
+This section provides detailed lists of changes by :ref:`application <applications>`.
+
+Asset Tracker v2
+----------------
+
+* Updated the Wi-Fi configurations to reduce the RAM usage by about 25 kB for an nRF91 Series DK and 12 kB for the Thingy:91 X.
+
+Connectivity bridge
+-------------------
+
+* Updated the handling of USB CDC ACM baud rate requests to make sure the baud rate is set correctly when the host requests a change.
+  This fixes an issue when using GNU screen with the Thingy:91 X.
+
+Matter bridge
+-------------
+
+* Added:
+
+  * Support for the ``UniqueID`` attribute in the Bridged Device Basic Information cluster.
+  * Version 2 of the bridged device data scheme containing ``UniqueID``.
+  * Kconfig options :ref:`CONFIG_BRIDGE_MIGRATE_PRE_2_7_0 <CONFIG_BRIDGE_MIGRATE_PRE_2_7_0>` and :ref:`CONFIG_BRIDGE_MIGRATE_VERSION_1 <CONFIG_BRIDGE_MIGRATE_VERSION_1>` to enable migration from older data schemes.
+
+nRF5340 Audio
+-------------
+
+* Updated:
+
+  * The documentation for :ref:`nrf53_audio_app_building` with cross-links and additional information, based on user feedback.
+  * The calculation in ``audio_datapath.num_blks_in_fifo`` to consider wrapping.
+
+nRF Desktop
+-----------
+
+* Updated:
+
+  * The :ref:`nrf_desktop_settings_loader` to make the :ref:`Zephyr Memory Storage (ZMS) <zephyr:zms_api>` the default settings backend for all board targets that use the MRAM technology.
+    As a result, all :ref:`zephyr:nrf54h20dk_nrf54h20` configurations were migrated from the NVS settings backend to the ZMS settings backend.
+  * :ref:`nrf_desktop_watchdog` by adding the :ref:`zephyr:nrf54h20dk_nrf54h20` release configuration.
```

Comment:
Hi @peknis, please move this entry back to the changelog, it is relevant for 2.9-nRF54H20.